modules/base/src/components/Settings.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="settings">
+    <OptionsView
+      :configLayers="configLayers"
+      :configOptions="configOptions"
+      :activeReleaseLevel="activeReleaseLevel"
+      :activeExpertiseLevel="activeExpertiseLevel"
+    />
+  </div>
+</template>
+
+<script>
+import OptionsView from "./options/OptionsView.vue";
+
+export default {
+  name: "Settings",
+  components: {
+    OptionsView
+  },
+  data() {
+    return {
+      configLayers: {}
+    };
+  },
+  computed: {
+    configOptions() {
+      return this.$parent.configDB.records;
+    },
+    activeReleaseLevel() {
+      return this.$parent.activeReleaseLevel;
+    },
+    activeExpertiseLevel() {
+      return this.$parent.activeExpertiseLevel;
+    }
+  },
+  methods: {
+    setConfig(key, value) {
+      // TODO
+      console.log("setting " + key + " to " + value); // eslint-disable-line
+    },
```

Comment:
Seems like this have been forgotten.

Reply:
Yes, this will be used when the App Settings are integrated.